toolkits/springernature/packages/springernature-header/view/header.hbs

```diff
@@ -0,0 +1,36 @@
+<header class="c-header">
+    <div class="c-header__inner">
+        <div class="c-header__content">
+            <div class="u-flex-wrap">
```

Comment:
Don't rely on utility classes in a component - if there is more than one style definition there should be a mixin that comes with `brand-context`, or (as in this case) if it is just one-line incorporate it into your component CSS (using a `c-` class). Components should be standalone modules.